src/Microsoft.Health.Fhir.SqlServer/Features/Storage/Registry/SqlServerSearchParameterStatusDataStore.cs

```diff
@@ -49,7 +49,7 @@ internal class SqlServerSearchParameterStatusDataStore : ISearchParameterStatusD
         public async Task<IReadOnlyCollection<ResourceSearchParameterStatus>> GetSearchParameterStatuses()
         {
             // If the search parameter table in SQL does not yet contain status columns
-            if (_schemaInformation.Current < 4)
+            if (_schemaInformation.Current < 6)
```

Comment:
Was this just the wrong value?

Reply:
Might be a good idea to have a constant or add a helper method to do the check since you do it in two places.

Reply:
Yep, this value was just outdated. Good idea, I added it as a constant which I can also use in `SqlServerFhirModel` 😎 (please see 470683c).